doc/nrf/releases_and_maturity/migration/migration_guide_3.0.rst

```diff
@@ -36,7 +36,15 @@ Libraries
 
 This section describes the changes related to libraries.
 
-|no_changes_yet_note|
+nRF Cloud library
+-----------------
+
+.. toggle::
+
+   For applications and samples using the :ref:`lib_nrf_cloud` library:
+
+   * It will be required to enable :kconfig:option:`CONFIG_NRF_CLOUD` Kconfig option in order to access the nRF Cloud libraries.
+     This option is now disabled by default to prevent the unintended inclusion of nRF Cloud Kconfig variables in non-nRF Cloud projects, addressing a previous issue.
```

Comment:
```suggestion
   * You must set the :kconfig:option:`CONFIG_NRF_CLOUD` Kconfig option to access the nRF Cloud libraries.
     This option is now disabled by default to prevent the unintended inclusion of nRF Cloud Kconfig variables in non-nRF Cloud projects, addressing a previous issue.
```